src/components/sections/DmEditSidebar.vue

```diff
@@ -1,8 +1,30 @@
 <template>
   <v-navigation-drawer class="px-2" floating :width="450" color="transparent">
     <v-card class="elevation-0">
+      <v-text-field class="pt-3 pl-3 mr-2" v-model="search" prepend-inner-icon="mdi-magnify" clearable label="Asignatura"
+        type="text" variant="outlined" :append-icon="'mdi-cog-outline'" @click:append="showFilter = !showFilter"
+        @click:clear="clearSearch">
+      </v-text-field>
+      <v-expand-transition v-show="showFilter">
+        <v-card-text class="text-subtitle-1">
+          Semestre
+          <v-row>
+            <v-col cols="7" class="mt-1">
+              <v-autocomplete v-model="selectedLevel" chips closable-chips multiple clearable :items="levelOptions"
+                label="Seleccione el nivel" item-title="name" item-value="value"></v-autocomplete>
```

Comment:
This section has 2 label. Use just one for the autocomplete component or the one outside it.